demos/ids-error-page/example.html

```diff
@@ -0,0 +1,18 @@
+<ids-layout-grid auto="true">
+  <ids-layout-grid-cell>
+    <ids-text font-size="12" type="h1">IDS Error Page</ids-text>
+  </ids-layout-grid-cell>
+</ids-layout-grid>
+
+<ids-layout-grid auto="true">
+  <ids-layout-grid-cell>
+    <ids-error-page
+      id="error-page-1"
+      visible="true"
+      icon="empty-error-loading"
+      label="Access Denied"
+      description="SunSystems Configuration Manager access has been denied."
```

Comment:
```suggestion
      description="Configuration Manager access has been denied."
```

Just thinking about removing specific names